scripts/ethbuild.sh

```diff
@@ -223,7 +238,12 @@ do
 			exit 1
 		fi
 
-		make -j${MAKE_CORES}
+		if [[ $USEXCODE -eq 0 ]]; then
+			make -j${MAKE_CORES}
+		else
+			xcodebuild
```

Comment:
So do you think this PR is good to be merged?

The main reason is using xcode allows developers to leverage some debugging features in Xcode. Using raw lldb on Mac OSX is not quite fun. :)